assets/base/i18n/ru.json

```diff
@@ -80,10 +80,10 @@
   "integrations.add": "Добавить",
   "intro_messages.anyMember": " Любой участник может зайти и читать этот канал.",
   "intro_messages.beginning": "Начало {name}",
-  "intro_messages.creator": "{name} - {type}, созданный {creator} {date}",
-  "intro_messages.creatorPrivate": "{name} - {type}, созданный {creator} {date}",
+  "intro_messages.creator": "Канал: {name}, создан {creator} {date}",
+  "intro_messages.creatorPrivate": "Приватный канал: {name}, созданный {creator} {date}",
   "intro_messages.group_message": "Начало истории групповых сообщений с участниками. Размещённые здесь сообщения и файлы не видны за пределами этой области.",
-  "intro_messages.noCreator": "{name} - {type}, созданный {date}",
+  "intro_messages.noCreator": "Канал: {name}, созданный {date}",
```

Comment:
Is there a special reason that the placeholder {type} is not used in the translation?

Reply:
@marianunez the original message was changed to `This is the start of the {name} channel, created on {date}.`